hw2.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+// HW2. По темі ‘Об'єкти в деталях’ + функції
+// Домашнє завдання по темі "Об'єкти"
+
+
+// Задача про обчислення різниці часу
+
+function durationBetweenDates (start, end, dimension ) {
+    if (dimension == 'days') {
```

Comment:
```suggestion
    if (dimension === 'days') {
```
І тут і далі, не використовуйте не строгу рівність. Тільки `===` та `!==`